docs/CODE_CONVENTIONS.md

```diff
@@ -71,22 +71,23 @@ class ModelClass:
 
 ```
 
+## Quotes
+You should use only double quotes "" not never single quotes.
```

Comment:
Still think that we should use single quotes for type hints.

You are right about single consistent style for every string. PEP8 declares:
https://www.python.org/dev/peps/pep-0008/#string-quotes
`In Python, single-quoted strings and double-quoted strings are the same. This PEP does not make a recommendation for this. Pick a rule and stick to it.`

but I don't think that they considered type hints in here.

Also, as far as I see, no one uses double quotes for type hints.
It seems like it is unspoken agreement.

Also, I'd like to separate visually strings that means types and strings which are used as... printable strings.

If you disagree with above statements, just leave a comment here, and we'll stick to rule with double-quotes.